packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryConfirmKey/index.tsx

```diff
@@ -84,8 +85,19 @@ const AccountRecoveryConfirmKey = (_: RouteComponentProps) => {
         await account.getRecoveryKeyBundle(accountResetToken, recoveryKey, uid);
 
       logViewEvent('flow', `${viewName}.success`, REACT_ENTRYPOINT);
-      navigate('/account_recovery_reset_password', {
-        state: { accountResetToken, email, recoveryData, recoveryKeyId },
+
+      // FOLLOW-UP: Get values by decoding recovery data.
+      const decode = (_data: string) => ({
+        kB: randomBytes(64).toString('hex'),
+      });
+      const { kB } = decode(recoveryData);
+
+      navigate(`/account_recovery_reset_password${window.location.search}`, {
```

Comment:
@millsoper @vpomerleau I had to add this to get the following page to actually work.